es/news/_posts/2020-03-19-json-dos-cve-2020-10663.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: news_post
+title: "CVE-2020-10663: Vulnerabilidad de Creación Insegura de Objetos en JSON (Corrección adicional)"
+author: "mame"
+translator: vtamara
```

Comment:
```suggestion
translator: "vtamara"
```
@vtamara Hello!
I don't insist but you can add quotation marks alike for author. Can you change CVE-2020-10663 name for the PR title and commit message please?